posts/2024-07-17_test4-live/README.md

```diff
@@ -0,0 +1,48 @@
+---
+publication_date: 2024-07-17T00:00:00Z
+slug: test4-live
+tags: [gnoland, ecosystem, govdao, test4]
+authors: [Kouteki]
+---
+
+# Blockchain Testnet Launched: Test4 Now Live
+
+We are excited to announce the launch of our blockchain testnet, Test4. This milestone marks a significant leap in our journey towards the gno.land mainnet, bringing new capabilities and stability to our development environment.
+
+## Key Features of Test4
+
+### Transition to a Decentralized Network
+
+Test4 is our first step towards a fully decentralized blockchain network. Unlike earlier stages, it supports multiple validator nodes, enhancing security and resilience. The initial validator set consists of:
+
+- Core engineering team
+- DevX team
+- Onbloc
+
+Which brings us to the second major feature, the GovDAO.
+
+### GovDAO
+
+The first iteration of the GovDAO enables on-chain voting and validator management. The initial GovDAO members are gno.land team members, major contributors and teams already building dapps on gno.land. The first voting proposals will be to add additional 3rd party validators:
+
+- Berty
+- Teritori
+- IrreverentSimplicity
+
+Our plan is to keep evolving the GovDAO mechanism, with the goal of setting up a decentralized entity that incentivizes members to be active and contribute to the gno.land project.
+
+### A Stable Environment for dApp Development
+
+Test4 is the first permanent and stable testnet for Gno.land, designed to avoid breaking changes. To help developers use it to build and test dApps, we've added Test4 to the [Faucet Hub](https://faucet.gno.land/); you're able to obtain test tokens and experiment without real consequences.
+
+We've also collaborated with Onbloc to update [Adena](https://www.adena.app/), which now supports Test4 out of the box. Onbloc has also provided [GnoScan](https://gnoscan.io/), a transaction explorer to aid in smart contract development and troubleshooting.
+
+### Namespaces
+
+Namespaces provide users with the exclusive capability to publish contracts under their designated namespaces, similar to GitHub's user and organization model. Or simply put, you'll be able to claim a username and use it as your on-chain identity. Learn more about it in the [gnoland docs](https://docs.gno.land/concepts/namespaces/).
+
+## What's Next?
+
+We've launched Test4 a few weeks ahead of the schedule - we'll be heads down the next few weeks fixing bugs and patching thingsS. There's still a lot of work left before we get to a mainnet release candidate. That's where you come in: play around with Test4, break it, and open issues on [GitHub](https://github.com/gnolang/gno) to let us know what needs to be fixed.
+
+Only together we'll be able to build gno.land the right way. So let's get started, gnomies!
```

Comment:
> gnomies

lol

Maybe switch to `gnomes` for a more official tone.

Reply:
That one's intentional :)